.npmignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Having a whitelist `files` inside `package.json` supercedes having a `.npmignore`, so we can remove this file.